tensordict/tensordict.py

```diff
@@ -2348,6 +2348,24 @@ def masked_fill(self, mask: Tensor, value: float | bool) -> TensorDictBase:
         """
         raise NotImplementedError
 
+    def masked_select_(self, mask: Tensor) -> TensorDictBase:
+        """Masks all tensors of the TensorDict.
+
+        Args:
+            mask (torch.Tensor): boolean mask to be used for the tensors.
+                Shape must match the TensorDict batch_size.
+
+        Examples:
+            >>> td = TensorDict(source={'a': torch.zeros(3, 4)},
+            ...    batch_size=[3])
+            >>> mask = torch.tensor([True, False, False])
+            >>> td.masked_select_(mask)
+            >>> td.get("a")
+            tensor([[0., 0., 0., 0.]])
+
+        """
+        raise NotImplementedError
```

Comment:
Since we implement it for one class only, let's raise a more proper error message, something like
```suggestion
        raise NotImplementedError(f"class {type(self)} does not implement masked_select_. Convert to a `TensorDict` instance first. If this is needed, open an issue on tensordict's repository.")
```

Reply:
Thanks